userscript/lovely-forks.user.js

```diff
@@ -0,0 +1,501 @@
+// ==UserScript==
+// @name         Lovely Forks
+// @namespace    musically-ut
+// @version      2.6.2
+// @description  Show notable forks of GitHub projects.
+// @homepage     https://github.com/musically-ut/lovely-forks
+// @icon         https://github.com/musically-ut/lovely-forks/raw/master/userscript/icon.png
```

Comment:
I'd rather only have a single location for the icon, in case I ever choose to change it. :)

Was there a reason to upload another version that I don't know of? e.g. compatibility with some user-script extensions that I do not know about?